tools/picard.wdl

```diff
@@ -56,7 +56,7 @@ task validate_bam {
         Boolean summary_mode = false
         Boolean index_validation_stringency_less_exhaustive = false
         Boolean ignore_warnings = true
-        String output_filename = basename(bam, ".bam") + ".validation_report.txt"
+        String output_filename = basename(bam, ".bam") + ".ValidateSamFile.txt"
```

Comment:
Why this change? Is this what `multiqc` expects? @a-frantz

Reply:
Yes. There were a bunch of our results not showing up in MultiQC, so I went through everything and checked what it was expecting. I'm testing it now, but the docs say that should be the file name extension.

Reply:
Good deal.